test/board_test.exs

```diff
@@ -17,37 +17,37 @@ defmodule BoardTest do
             board_1 = Board.update(Board.empty_board, 0, :player_one)
             assert board_1 == %{0 => :player_one}
     
-            board_2 = Board.update(board_1, 1, :player_two)
+            _board_2 = Board.update(board_1, 1, :player_two)
```

Comment:
Not sure what you're asserting against if you renamed to `_board_2`?

Reply:
<img width="511" alt="Screen Shot 2019-04-24 at 4 28 23 PM" src="https://user-images.githubusercontent.com/21062007/56695163-05410280-66ae-11e9-941d-df82f079a2fb.png">

I get this warning without the underscore

Reply:
Try running the test without the `assert board_1 == %{0 => :player_one}` assertions. You don't need to repeatedly assert that after the `"one player can make one mark on the board"` test.

Reply:
I still get the same warning :(